pkg/controller/promotionreconciler/testdata/ocp.413.grafana.istag.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: image.openshift.io/v1
+generation: 146
+image:
+  dockerImageLayers:
+  - mediaType: application/vnd.docker.image.rootfs.diff.tar.gzip
+    name: sha256:eac1b95df832dc9f172fd1f07e7cb50c1929b118a4249ddd02c6318a677b506a
+    size: 83303635
+  - mediaType: application/vnd.docker.image.rootfs.diff.tar.gzip
+    name: sha256:47aa3ed2034c4f27622b989b26c06087de17067268a19a1b3642a7e2686cd1a3
+    size: 1789
+  - mediaType: application/vnd.docker.image.rootfs.diff.tar.gzip
+    name: sha256:4faf6d0e05df8c3aac88f12f0fdf5d967121cff63787bfb9c716947312f7ba6b
+    size: 17001180
+  - mediaType: application/vnd.docker.image.rootfs.diff.tar.gzip
+    name: sha256:abdd42b48a77fa84a6cf84cc48c94ae607b6699d1b7b0d5a37cedc6ab3ecf382
+    size: 474512
+  - mediaType: application/vnd.docker.image.rootfs.diff.tar.gzip
+    name: sha256:9986306e3f5dc5d82cc5d43e770e11288ddb61039fe39467da02f17197e82d0a
+    size: 12076325
+  - mediaType: application/vnd.docker.image.rootfs.diff.tar.gzip
+    name: sha256:8dddce76952be363bffcc39bcf59d790917a9f6bc61c3ce8763bac47b7d4a897
+    size: 72696466
+  dockerImageManifestMediaType: application/vnd.docker.distribution.manifest.v2+json
+  dockerImageMetadata:
+    Architecture: amd64
+    Config:
+      Entrypoint:
+      - /run.sh
+      Env:
+      - __doozer=merge
+      - BUILD_RELEASE=202202280820.p0.gf17f552.assembly.art3171
+      - BUILD_VERSION=v4.11.0
+      - OS_GIT_MAJOR=4
+      - OS_GIT_MINOR=11
+      - OS_GIT_PATCH=0
+      - OS_GIT_TREE_STATE=clean
+      - OS_GIT_VERSION=4.11.0-202202280820.p0.gf17f552.assembly.art3171-f17f552
+      - SOURCE_GIT_TREE_STATE=clean
+      - OS_GIT_COMMIT=f17f552
+      - SOURCE_DATE_EPOCH=1617214817
+      - SOURCE_GIT_COMMIT=f17f5523faca5bd928d5cc65484a66b20f04f366
+      - SOURCE_GIT_TAG=f17f5523
+      - SOURCE_GIT_URL=https://github.com/openshift/ocp-build-data
```

Comment:
If this is a real example of one of the failure cases, isn't this a residue of the base image (I suppose)?

https://github.com/openshift/ocp-build-data/commit/f17f5523faca5bd928d5cc65484a66b20f04f366

While for this controller we want to track changes to the component repository.

Reply:
https://docs.openshift.com/container-platform/4.11/cicd/builds/managing-build-output.html#builds-output-image-environment-variables_managing-build-output

I cannot see the difference between label and the env var above.
Let me ask the build team.

Reply:
Didi it ever worked?
I think we should use only `SOURCE_GIT_COMMIT`.

Reply:
We actually have both:

```console
oc get istag -n ci ci-operator:latest -o yaml | grep -i commit
        io.openshift.build.commit.author: ""
        io.openshift.build.commit.date: ""
        io.openshift.build.commit.id: f80a6fc6d9b4f140176863a1ada1e98d9c9f11dc
        io.openshift.build.commit.message: ""
        io.openshift.build.commit.ref: master
        io.openshift.build.commit.author: ""
        io.openshift.build.commit.date: ""
        io.openshift.build.commit.id: f80a6fc6d9b4f140176863a1ada1e98d9c9f11dc
        io.openshift.build.commit.message: ""
        io.openshift.build.commit.ref: master
```

We need to tell how to use them.

Reply:
Consulting the build team.
https://coreos.slack.com/archives/C014MHHKUSF/p1668613432332029

Reply:
/hold

blocked by https://issues.redhat.com/browse/OCPBUGS-3785